resolwe/process/fields.py

```diff
@@ -358,9 +353,21 @@ def to_python(self, value):
 
     def to_output(self, value):
         """Convert value to process output format."""
-        return json.loads(
-            resolwe_runtime_utils.save_file(self.name, value.path, *value.refs)
-        )
+        data = {"file": value.path}
```

Comment:
You should also check that this file exists?

Reply:
It depends what do we want. I believe validating that the file exists at this point is pointless for two reasons:
1. The file can later be remove by the process so validation does not really help.
2. The existence of files is done anyway when data object is validated.

But it is true that errors in scripts are probably discovered sooner if file validation is done also at this point.